xwiki-platform-core/xwiki-platform-web/overlays/org.xwiki.platform.xwiki-platform-index-tree-war-9.11.8-SNAPSHOT.info

```diff
@@ -0,0 +1,3 @@
+1539692341000
```

Comment:
I don't think you should include that file.

Reply:
Indeed. Shouldn't it be in the gitignore @vmassol ?

Reply:
Indeed. I don't even understand what is this file, never seen this.

Reply:
I saw the folder "overlays" several times, deleting it each time, except this one or I went a bit fast with the git add.

Reply:
Indeed. Now I've removed it and rebuilt platofrm-web and it didn't reappear. I thought it was the war plugin workDirectory but the default value seems ok: https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#workDirectory

Maybe it appears only when run in a reactor.

Reply:
I tried `mvn -pl org.xwiki.platform:xwiki-platform-web clean install` from the root and no overlays dir either

Reply:
Apparently the version is 9.11.8. It's certainly related to the `mvn install` done on an old branch.

Reply:
I sometimes have the folder too, I think we should simply do as @surli said and add a rule into the .gitignore file.

Reply:
.gitignore would certainly not do any harm, at worst it's just useless